tools/runners/proxy_runner.py

```diff
@@ -18,6 +20,10 @@ def setup_parser(subparsers: ArgumentParser) -> ArgumentParser:
     contract_group = contract_parser.add_subparsers()
     add_upgrade_command(contract_group, upgrade_proxy_dex_contracts)
 
+    transaction_parser = subgroup_parser.add_parser('transaction', help='proxy dex transaction commands')
```

Comment:
use the standard naming that Claudiu defined: `generate-transactions` instead of `transaction`